data/catalogs/deltares_data.yml

```diff
@@ -1531,249 +1661,268 @@ simard_v2011:
+        \ Liedekerke, M., Jones, A., Montanarella, L., 2012. European soil data centre:\
+        \ Response to european policy support  and public data requirements. Land\
+        \ Use Policy 29 (2), 329\xC3\xA2\xE2\u201A\xAC\xE2\u20AC\u0153338. \n"
+      paper_doi: 10.1371/journal.pone.0169748
+      paper_ref: Hengl et al. (2017)
+      source_license: ODbL
+      source_url: https://www.isric.org/explore/soilgrids/faq-soilgrids-201
+    rename:
+      BLDFIE_M_sl1_250m_ll: bd_sl1
+      BLDFIE_M_sl2_250m_ll: bd_sl2
+      BLDFIE_M_sl3_250m_ll: bd_sl3
+      BLDFIE_M_sl4_250m_ll: bd_sl4
+      BLDFIE_M_sl5_250m_ll: bd_sl5
+      BLDFIE_M_sl6_250m_ll: bd_sl6
+      BLDFIE_M_sl7_250m_ll: bd_sl7
+      CLYPPT_M_sl1_250m_ll: clyppt_sl1
+      CLYPPT_M_sl2_250m_ll: clyppt_sl2
+      CLYPPT_M_sl3_250m_ll: clyppt_sl3
+      CLYPPT_M_sl4_250m_ll: clyppt_sl4
+      CLYPPT_M_sl5_250m_ll: clyppt_sl5
+      CLYPPT_M_sl6_250m_ll: clyppt_sl6
+      CLYPPT_M_sl7_250m_ll: clyppt_sl7
+      ORCDRC_M_sl1_250m_ll: oc_sl1
+      ORCDRC_M_sl2_250m_ll: oc_sl2
+      ORCDRC_M_sl3_250m_ll: oc_sl3
+      ORCDRC_M_sl4_250m_ll: oc_sl4
+      ORCDRC_M_sl5_250m_ll: oc_sl5
+      ORCDRC_M_sl6_250m_ll: oc_sl6
+      ORCDRC_M_sl7_250m_ll: oc_sl7
+      PHIHOX_M_sl1_250m_ll: ph_sl1
+      PHIHOX_M_sl2_250m_ll: ph_sl2
+      PHIHOX_M_sl3_250m_ll: ph_sl3
+      PHIHOX_M_sl4_250m_ll: ph_sl4
+      PHIHOX_M_sl5_250m_ll: ph_sl5
+      PHIHOX_M_sl6_250m_ll: ph_sl6
+      PHIHOX_M_sl7_250m_ll: ph_sl7
+      SLTPPT_M_sl1_250m_ll: sltppt_sl1
+      SLTPPT_M_sl2_250m_ll: sltppt_sl2
+      SLTPPT_M_sl3_250m_ll: sltppt_sl3
+      SLTPPT_M_sl4_250m_ll: sltppt_sl4
+      SLTPPT_M_sl5_250m_ll: sltppt_sl5
+      SLTPPT_M_sl6_250m_ll: sltppt_sl6
+      SLTPPT_M_sl7_250m_ll: sltppt_sl7
+      SNDPPT_M_sl1_250m_ll: sndppt_sl1
+      SNDPPT_M_sl2_250m_ll: sndppt_sl2
+      SNDPPT_M_sl3_250m_ll: sndppt_sl3
+      SNDPPT_M_sl4_250m_ll: sndppt_sl4
+      SNDPPT_M_sl5_250m_ll: sndppt_sl5
+      SNDPPT_M_sl6_250m_ll: sndppt_sl6
+      SNDPPT_M_sl7_250m_ll: sndppt_sl7
+      SoilThickness_250m_ll: soilthickness
+      TAXOUSDA_250m_ll: tax_usda
+    unit_mult:
+      bd_sl1: 0.001
+      bd_sl2: 0.001
+      bd_sl3: 0.001
+      bd_sl4: 0.001
+      bd_sl5: 0.001
+      bd_sl6: 0.001
+      bd_sl7: 0.001
+      oc_sl1: 0.1
+      oc_sl2: 0.1
+      oc_sl3: 0.1
+      oc_sl4: 0.1
+      oc_sl5: 0.1
+      oc_sl6: 0.1
+      oc_sl7: 0.1
+      ph_sl1: 0.1
+      ph_sl2: 0.1
+      ph_sl3: 0.1
+      ph_sl4: 0.1
+      ph_sl5: 0.1
+      ph_sl6: 0.1
+      ph_sl7: 0.1
+  - version: 2.0
+    path: soil/soilgrids_v2.0/*/*_mean.vrt
+    meta:
+      paper_doi: https://doi.org/10.5194/soil-2020-65
+      paper_ref: de Sousa et al. (2020)
+      source_license: CC BY 4.0
+      source_url: https://www.isric.org/explore/soilgrids/faq-soilgrids
+      processing_script: soil/soilgrids_v2.0/src/preprocess_soilgrids_v2020.ipynb
+      processing_notes: soil/soilgrids_v2.0/src/readme.txt
```

Comment:
Maybe mention that soilthickness is taken from version 1 as unavailable in version 2 ?